sdk/python/feast/permissions/client/arrow_flight_auth_interceptor.py

```diff
@@ -0,0 +1,39 @@
+import pyarrow.flight as fl
+
+from feast.permissions.auth.auth_type import AuthType
+from feast.permissions.auth_model import AuthConfig
+from feast.permissions.client.auth_client_manager import get_auth_client_manager
+
+
+class FlightBearerTokenInterceptor(fl.ClientMiddleware):
+    def __init__(self, auth_config: AuthConfig):
+        super().__init__()
+        self.auth_config = auth_config
+
+    def call_completed(self, exception):
+        pass
+
+    def received_headers(self, headers):
+        pass
+
+    def sending_headers(self):
+        auth_client_manager = get_auth_client_manager(self.auth_config)
```

Comment:
Cant this use `create_auth_header` from `feast/permissions/client/utils.py` like in other clients ?